src/index.html

```diff
@@ -6,9 +6,61 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
+
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="site-header">
+      <div class="inner-wrapper">
+        <div class="logo_wrapper">
+          <a href="#home" class="logo">
+            <img src="images/logo.png" alt="site logo">
+          </a>
+        </div>
+
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#apple"
+                class="nav_link is-active">
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav_link">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__item">
              <a
                href="#apple"
                class="nav_link is-active">
                Apple
              </a>
            </li>
            
            <li class="nav__item">
              <a href="#samsung" class="nav_link">Samsung</a>
            </li>
```

Add empty line between multiple sibling blocks of html.

Fix all such cases